src/routes/Devilstable.svelte

```diff
@@ -71,45 +64,101 @@
 		}
 		return fractionData;
 	});
+	/** @type {{label: string, values: boolean[]}[]} */
 	let boolData = $derived([
-		...noFractions.map((d) => {
-			/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+		...inputChipValueLabels.map((d) => {
+			const info = data.find((c) => c.label === d);
+			if (info) {
+				/** @type {boolean[]} */ const values = new Array(DATA_MAX).fill(false);
+				info?.values.forEach(([start, end]) => {
+					for (let i = start; i <= end; i++) {
+						// Adjust for 0-indexed array
+						values[i - 1] = true;
+					}
+				});
 
-			d.values.forEach(([start, end]) => {
-				for (let i = start; i <= end; i++) {
-					// Adjust for 0-indexed array
-					values[i - 1] = true;
+				return {
+					label: d,
+					values
+				};
+			} else {
+				if (d === 'fr') {
+					return allFractionData;
+				} else {
+					return {
+						label: d,
+						values: new Array(DATA_MAX).fill(true)
+					};
 				}
-			});
-
-			return {
-				label: d.label,
-				values
-			};
-		}),
-		fractionData
+			}
+		})
 	]);
+	let selection = $state({
+		start: 1,
+		end: 100
+	});
+	let detailData = $derived(
+		boolData.map((d) => {
+			return { label: d.label, values: d.values.slice(selection.start - 1, selection.end) };
+		})
+	);
 </script>
 
+<div class="card p-4 variant-filled-primary max-w-lg" data-popup="popupClick">
+	<p>
+		Um Textzeugen und Fragmente zu entfernen, klicken sie bitte einfach auf die grau hinterlegten
+		Kasten.
+	</p>
+	<p>
+		Um einzelne Fragmente hinzuzufügen geben sie <i>fr</i> gefolgt vom Index des Fragments ein. Z.B.
+		<i>fr32</i>.
+	</p>
+	<div class="arrow variant-filled-primary"></div>
+</div>
+<div
+	class="container mx-auto mb-6 flex flex-wrap md:flex-nowrap gap-4"
+	use:popup={{ event: 'focus-click', placement: 'top', target: 'popupClick' }}
+>
+	<InputChip
+		whitelist={[...allowStringsWithoutFragments, ...fractions.map((f) => f.label)]}
+		bind:value={inputChipValues}
+		placeholder="Textzeuge / Fragment hinzufügen..."
+		name="Inputchips"
+		allowUpperCase
+	/>
+	<div class="btn-group md:btn-group-vertical variant-filled h-min m-auto">
+		<button
+			class="btn variant-filled"
+			onclick={() => {
+				inputChipValues = allowStringsWithoutFragments;
+			}}
+		>
+			zurücksetzen
+		</button>
+		<button
+			class="btn variant-filled"
+			onclick={() => {
+				inputChipValues = [];
+			}}
+		>
+			leeren
+		</button>
+	</div>
+</div>
 <Brush
 	width={mobile ? width : brushDimension}
 	height={mobile ? brushDimension : height}
-	data={boolData}
-	brushE={(e) => (selection = e)}
+	data={boolData.filter((d) => d.label !== 'Fassung')}
```

Comment:
hardcoding 'Fassung' is dangerous. Currently renaming 'Fassung' requires 3 edits in Details.svelte and Devilstable.svelte which is too much.

Reply:
any chance to add this to `sigla.json`, just to have the outsourced?

Reply:
Adding it to the sigla-meta-information wouldn't help. (I deleted the sigla.json present in the $lib. It was a dummy file.

I abstracted the string into a constant in constants.js.
Is that ok like that? See changes in aed9718879abff904ea0ecd35d0c4581b44fa357

Reply:
oh, I didn't notice it's a dummy file, I found it via global search. 
I'm happy with your solution, `constants.js` is an excellent place.